packages/hint-font-relative-units/README.md

```diff
@@ -0,0 +1,66 @@
+# font-relative-units (`@hint/hint-font-relative-units`)
+
+Ensure font styles use relative units instead of pixels.
+
+## Why is this important?
+
+If website stylesheets define certain styles in pixels, they cannot be easily overridden by custom stylesheets and browser plugins. Some people benefit from altering font size, line height, and letter spacing to make text easier to read.
```

Comment:
Our linters are going to complain if the length of a line in markdown is more than 80 characters.

If you run `yarn test` from your new hint to run all linters.

Reply:
I think we should reword the "why is this important" section. 
Pixels are not a relative unit and do not scale, but that doesn't necessarily mean pixels are hard to override. Because pixels aren't in a scalable unit, when users who need to adjust a web page for readability, they don't actually scale to increase or decrease with page zoom, disadvantaging users who need to alter those settings.

Reply:
@ststimac it’s less for zoom (most browsers will actually zoom pixel values as well as relative units) and more for changing the default browser font size or user stylesheets (or plugins that work by adding custom css) that set top-level font size, line height, etc.

How would you feel about a change to the wording along the lines of “If website stylesheets define certain styles in pixels, they do not reflect changes to root-level font size”?

Reply:
That sounds good!